Doc/using/configure.rst

```diff
@@ -289,6 +289,143 @@ General Options
 
    .. versionadded:: 3.13
 
+.. cmdoption:: PKG_CONFIG
+
+   Path to ``pkg-config`` utility.
+
+.. cmdoption:: PKG_CONFIG_LIBDIR
+.. cmdoption:: PKG_CONFIG_PATH
+
+   ``pkg-config`` options.
+
+
+C Compiler Options
+------------------
+
+.. cmdoption:: CC
+
+   C compiler command.
+
+.. cmdoption:: CFLAGS
+
+   C compiler flags.
+
+.. cmdoption:: CPP
+
+   C preprocessor command.
+
+.. cmdoption:: CPPFLAGS
+
+   C preprocessor flags, e.g. ``-I<include dir>``.
+
+
+Linker Options
+--------------
+
+.. cmdoption:: LDFLAGS
+
+   Linker flags, e.g. ``-L<library directory>``.
+
+.. cmdoption:: LIBATOMIC
+
+   Linker flags when ``cpython/pyatomic.h`` header file is used.
+
+   Default: ``LIBATOMIC='-latomic'`` if ``libatomic`` is needed, or
+   ``LIBATOMIC=''`` otherwise (also the default when Python is cross-compiled).
+
+   .. versionadded:: 3.13
+
+.. cmdoption:: LIBS
+
+   Libraries to pass to the linker, e.g. ``-l<library>``.
+
+.. cmdoption:: MACHDEP
+
+   Name for machine-dependent library files.
+
+
+Libraries Options
```

Comment:
How about _Third-party dependencies_ or _Options for third-party dependencies_?

Reply:
Go for "Options for third-party dependencies".